src/commonjs-index.js

```diff
@@ -0,0 +1,2 @@
+require = require("esm")(module);
```

Comment:
is this an intentional override of the `require` global?

Reply:
Hmm, I took this from [esm](https://github.com/standard-things/esm#getting-started), but if it messes up browserify, let's try to find a different solution. It looks like using a new name like esmRequire breaks the functionality, so I believe it's intentional, but hopefully not necessary.

Reply:
I asked the esm maintainers about this: https://github.com/standard-things/esm/issues/723

I also looked at how other people are using esm, and it looks like they are using the problematic `require` override. For instance, Mapbox uses it [like this](https://github.com/mapbox/mapbox-gl-js/blob/6c22a1720dac51eac1c898ca06b3e2f24dbe0810/test/query.test.js).

Reply:
The `require` function is not global in CommonJS and is a per-module instance variable. It's okay if you're uncomfortable or have linting rules preventing `require = require("esm")(module)` though _(it's just for doc example simplicity)_. You can always do something like:
```js
const esmRequire = require("esm")(module);

module.exports = esmRequire("./index.js");
```

For bundlers like webpack, parcel or browserify _(with the [`esmify`](https://www.npmjs.com/package/esmify) plugin)_ you'll want to add a `"module": "./index.js"` field to your `package.json` so that the bundlers bypass the CommonJS bridge file, `commonjs-index.js`,  and instead bundle the ESM module `./index.js`.

Reply:
Wow, thanks so much for such a quick response! This is incredibly helpful.

Reply:
I made the change and tested it and all seems to be well. @kumavis does this work for you?

Reply:
@katelynsills 
> I made the change and tested it and all seems to be well.

Awesome! 

👀 It looks like the `"module"` field is missing in the package.json for bundlers.

Reply:
Good catch, that is added!